src/java/org/apache/cassandra/cql3/restrictions/SingleColumnRestriction.java

```diff
@@ -873,6 +873,81 @@ public boolean isAnn()
         }
     }
 
+    /**
+     * A Bounded ANN Restriction is one that uses a distance as the limiting factor instead of a number of result
```

Comment:
At this level it's "similarity" not "distance" I think?

Reply:
Good point, I agree.